activiti-cloud-query/pom.xml

```diff
@@ -118,6 +118,9 @@
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
```

Comment:
why this?

Reply:
It blows up on Windows when trying to exec `mvn spring-boot:run`